packit/local_project.py

```diff
@@ -141,7 +141,8 @@ def __init__(
                 # since this can point to a fork
                 # .urls returns generator
                 self.git_url = list(self.git_repo.remote().urls)[0]
-                logger.debug("remote url of the repo is %s", self.git_url)
+                logger.debug(f"remote url of the repo is {self.git_url}")
+                self.namespace = get_namespace(self.git_url)
```

Comment:
The `get_namespace` is needed if we don't have `self.namespace` and know `self.git_repo`. Please, move it to the next `if`:

```suggestion
                change = True

            if self.git_repo and not self.namespace:
                self.namespace = get_namespace(self.git_url)
```